content/docs/actions.md

```diff
@@ -73,47 +82,81 @@ archive. If there are more steps, then one of them has to return the archive
- tarball can be unpacked (it tries to extract the directory name directly from the archive
- or uses the configured [`archive_root_dir_template`](/docs/configuration#archive_root_dir_template))
-3. It changes %version
+1. It replaces Source configured by
+   [`spec_source_id`](/docs/configuration/#spec_source_id) (default `Source0`)
+   with a local path to the generated archive.
 
-If you provide your own implementation, none of the above happens.
+2. It changes the first `%setup` (or `%autosetup`) macro in `%prep` and adds
+   `-n` so the generated tarball can be unpacked (it tries to extract the
+   directory name directly from the archive or uses the configured
+   [`archive_root_dir_template`](/docs/configuration#archive_root_dir_template)).
 
-For example a package may define multiple Sources. In such cases, the
-default implementation of `fix-spec-file` won't be able to update `%prep`
-correctly. You can write a simple shell script which will be run with the `cwd` set to
-the root of the repository and use `sed` to set the new
-Sources correctly, e.g. `sed -i my_specfile_path -e
-"s/https.*only-vendor.tar.xz/my_correct_tarball_path/"`
+3. It updates %version in the spec file.
 
+If you provide your own implementation, none of the above happens.
+
+For example a package may define multiple Sources. In such a case, the default
+implementation of `fix-spec-file` won't be able to update `%prep` correctly.
+You can instead use the `sed` program to set the new Sources correctly, e.g.
+```
+actions:
+  fix-spec-file:
+  # define one of the Source variables correctly
+  - sed -i my_specfile_path -e "s/https.*only-vendor.tar.xz/my_correct_tarball_path/"
+  # fill in %release as if packit would have done it
+  - bash -c "sed -i my_specfile_path -r \"s/Release:(\s*)\S+/Release:\1${PACKIT_RPMSPEC_RELEASE}%{?dist}/\""
+```
 
 ### Environment variables set by packit
 
-Additionally, packit sets a few env vars for specific actions
+Additionally, packit sets a few env vars for specific actions.
 
 **fix-spec-file**
 
-`PACKIT_PROJECT_VERSION` — current version of the project (coming from `git describe`)
-`PACKIT_PROJECT_COMMIT` — commit hash of the top commit
-`PACKIT_PROJECT_ARCHIVE` — expected name of the archive
+`PACKIT_PROJECT_VERSION` — current version of the project (coming from `git describe`)  
+`PACKIT_PROJECT_COMMIT` — commit hash of the top commit  
+`PACKIT_PROJECT_ARCHIVE` — expected name of the archive  
+`PACKIT_RPMSPEC_RELEASE` — value for spec file's `%release` field which packit would set  
 
 **create-archive**
 
 `PACKIT_PROJECT_VERSION` — current version of the project (coming from `git describe`)
 `PACKIT_PROJECT_NAME_VERSION` — current name and version of the project (coming from `git describe`)
 
+If you want to see the content of those variables, you can print using `echo`
+in the specific action:
+```
+actions:
+  fix-spec-file:
+  - bash -c "PACKIT_PROJECT_VERSION=${PACKIT_PROJECT_VERSION}"
```

Comment:
```suggestion
  - bash -c "echo PACKIT_PROJECT_VERSION=${PACKIT_PROJECT_VERSION}"
```

Reply:
my copy-pasting skills are atrocious this week :see_no_evil: